GANDLF/utils.py

```diff
@@ -316,10 +316,13 @@ def populate_header_in_parameters(parameters, headers):
     # initialize common parameters based on headers
     parameters["headers"] = headers
     # ensure the number of output classes for model prediction is working correctly
-    if len(headers["predictionHeaders"]) > 0:
+
+    if len(headers["predictionHeaders"]) > 1:
```

Comment:
Can you give me the rationale behind why the comparison value is `1` and not `0`?